deployments/k8s/helm/signalfx-agent/values.yaml

```diff
@@ -110,7 +110,7 @@ kubeletAPI:
 
 # Any values put in this object correspond to the 'collectd' config block of
 # the agent config
-collectd: {}
+collectd: {}  #we should be setting up the smart agent not the collectd agent.
```

Comment:
The Smart Agent runs collectd as a sub-process, which is what this config is about.  Normally the defaults for it are fine.